vector/src/main/java/im/vector/app/features/home/room/detail/timeline/item/AbsBaseMessageItem.kt

```diff
@@ -83,23 +113,36 @@ abstract class AbsBaseMessageItem<H : AbsBaseMessageItem.Holder> : BaseEventItem
                 reactionButton.isEnabled = reaction.synced
                 holder.reactionsContainer.addView(reactionButton)
             }
+            if (reactions.count() > MAX_REACTIONS_TO_SHOW) {
+                val showReactionsTextView = createReactionTextView(holder, 6)
```

Comment:
could be handy to have some named arguments for the hardcoded values for some extra context

Reply:
Actually, I'm changing this to a Style as it's shared with ReactionButton